tools/build_defs/cc/cc_import.bzl

```diff
@@ -128,10 +205,10 @@ def _cc_import_impl(ctx):
         name = ctx.label.name,
     )
 
-    return [CcInfo(
-        compilation_context = compilation_context,
-        linking_context = linking_context,
-    )]
+    if output_file:
+        _create_archive(ctx, feature_configuration, cc_toolchain, output_file, object_files)
```

Comment:
Part of me feels that `_create_archive` should be inlined; that declaring and defining an action is too important to bury behind this call. But I don't feel like I can insist.

Reply:
what about using a name like `_create_archive_action` to make this more obvious?